src/main/java/org/jasig/portlet/widget/links/ResourceLinksBaseController.java

```diff
@@ -0,0 +1,53 @@
+package org.jasig.portlet.widget.links;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.portlet.PortletConfig;
+import javax.portlet.PortletPreferences;
+import javax.portlet.PortletRequest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.portlet.context.PortletConfigAware;
+
+public class ResourceLinksBaseController implements PortletConfigAware {
+
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
📓 Lombok provides a nice helper to normalize log usage
Ref: https://projectlombok.org/features/log